packages/@react-spectrum/actiongroup/docs/ActionGroup.mdx

```diff
@@ -48,70 +48,71 @@ category: Buttons
+ActionGroups accepts a list of `children`, each with a `key` prop. It is required that the `children` are <TypeLink links={docs.links} type={docs.exports.Item} /> components from the [Collection Components](../react-stately/collections.html) API.
 
 However, a function that returns `Item` components is also accepted.
 
 ```tsx example
 const items = [
-  {children: 'React', name: 'React'},
-  {children: 'Add', name: 'Add'},
-  {children: 'Delete', name: 'Delete'}
+  {label: 'React', name: 'React'},
+  {label: 'Add', name: 'Add'},
+  {label: 'Delete', name: 'Delete'}
 ];
-<ActionGroup items={items}>
-  {item => <Item key={item.name}>{item.children}</Item>}
+<ActionGroup items={items} onAction={(key) => console.log(key)}>
+  {item => <Item key={item.name}>{item.label}</Item>}
 </ActionGroup>
 ```
 
+Each item needs to have a `key` prop so it can be identified when actioned upon by a user. See [Events](#events) for more information.
 
-Each item needs to have a `key` prop so it can be identified when actioned upon by a user.
-
-See [Events](#events) for more information.
-
-## Value
-
-ActionGroups are uncontrolled by default.
+### Accessibility
 
-Use `defaultSelectedKeys` to provide a default set of selected items. The value of the selected keys must match the `key` prop of the items.
+If no visible label is provided (e.g. an icon only) to each item, an alternative text label must be provided to identify the control for accessibility.
+This should be added using the `aria-label` prop to each Item.
 
 ```tsx example
-<ActionGroup selectionMode="single" defaultSelectedKeys={['list']}>
-  <Item key="grid">Grid view</Item>
-  <Item key="list">List view</Item>
-  <Item key="gallery">Gallery view</Item>
+<ActionGroup onAction={(key) => console.log(key)}>
+  <Item key="brush" aria-label="Brush"><Brush /></Item>
+  <Item key="select" aria-label="Select"><Select /></Item>
+  <Item key="regionSelect" aria-label="Select Region"><RegionSelect /></Item>
 </ActionGroup>
 ```
 
-The `selectedKeys` prop can be used to make the selected state controlled.
+### Internationalization
+
+In order to internationalize an ActionGroup, the strings of all child items should be localized.
+
+## Selection
+ActionGroup supports multiple selection modes. By default, selection is set to `none` however this can be changed using the `selectionMode` prop.
+Use `defaultSelectedKeys` to provide a default set of selected items (uncontrolled) and `selectedKeys` to set the selected items (controlled). The value of the selected keys must match the `key` prop of the items.
 
 ```tsx example
-<ActionGroup selectionMode="single" selectedKeys={['list']}>
+<ActionGroup selectionMode="single" defaultSelectedKeys={['list']} onSelectionChange={(selected) => console.log(selected)}>
```

Comment:
same. maybe display the list of selected items below similar to other component examples with selection?